apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -32,6 +32,20 @@
         Context :: #{}
                    ) -> {Status :: cowboy:http_status(), Headers :: list(), Body :: map()}.
 
+handle_request('PostKeyBlock', #{'KeyBlock' := Data}, _Context) ->
+    case aec_headers:deserialize_from_client(key, Data) of
+        {ok, Header} ->
+            KeyBlock = aec_blocks:new_key_from_header(Header),
+            case aec_conductor:post_block(KeyBlock) of
```

Comment:
I see that the key block does not have more info than the key block header, and - based on code inspection - the DB insertion of the posted block is ok (including recomputation of state trees with pruned items). Ok.